skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -103,10 +106,28 @@ private void setupAdapter() {
         recyclerView.setAdapter(formAdapter);
     }
 
-    @OnClick(R.id.bViewWifi)
-    public void viewWifiNetworks() {
-        Intent intent = new Intent(this, WifiActivity.class);
-        startActivity(intent);
+    @OnClick(R.id.bReceiveForms)
+    public void chooseReceivingMethods() {
+        Intent hotspotIntent = new Intent(this, HpReceiverActivity.class);
+        Intent bluetoothIntent = new Intent(this, BtReceiverActivity.class);
+
+        // choose different sending methods, TODO: needs refactor if we have a better UI/UX.
+        android.app.AlertDialog alertDialog = new android.app.AlertDialog.Builder(this)
+                .setTitle("Receive Options")
+                .setIcon(R.drawable.ic_help_outline)
+                .setItems(TRANSFER_OPTIONS, (DialogInterface dialog, int which) -> {
+                    switch (which) {
+                        case 0:
+                            startActivity(bluetoothIntent);
```

Comment:
create intent here and call 
`startActivity(intent);`
`finish();`
after switch block as this is a common statement for both the options of switch block

Reply:
Fixed.